### ISSUE TITLE
Soulbind malfunctions when binding an illusion to a hero

### ISSUE THREAD

**Issue by TheZett** [jeffhill/Dota2#72]
When an illusion is bound to another real hero, using any spell on the bound illusion will not cause the spell to get doubled onto the real hero.

The spell will only affect the illusion, the real hero does not get affected at all.

dev forum thread: https://dev.dota2.com/forum/bugs/gameplay-bugs/2269406-grimstroke-soulbind-does-not-double-spells-items-when-targeting-an-illusion

**Comment by nemoo2** [comment]
Is this similar to what's described here? 

https://old.reddit.com/r/DotA2/comments/thonzu/grimstroke_dark_portrait_soul_bind_interaction/

**Comment by TheZett** [comment]
Yes, this should be the same bug, where a spell is not duplicated when targeting an illusion.
